testscript/exe.go

```diff
@@ -42,11 +42,15 @@ func IgnoreMissedCoverage() {
 // code to pass to os.Exit. It's OK for a command function to
 // exit itself, but this may result in loss of coverage information.
 //
-// When Run is called, these commands will be available as
-// testscript commands; note that these commands behave like
-// commands run with the "exec" command: they set stdout
-// and stderr, and can be run in the background by passing "&"
-// as a final argument.
+// When Run is called, these commands will work when run via "exec".
```

Comment:
I think it should explicitly say that the commands are installed in the path and are hence available as normal commands (not just via `exec`) because that might not be entirely obvious to the casual reader.

How about something like this?

```
When Run is called, these commands are installed as regular commands in the shell
path, so can be invoked with "exec" or via any other command (for example a shell script).
```

Reply:
Good point, that's less ambiguous.